repos/spack_repo/builtin/packages/py_xarray/package.py

```diff
@@ -119,6 +130,23 @@ class PyXarray(PythonPackage):
         type=("build", "run"),
     )
 
+    # +viz
     depends_on("py-matplotlib", when="@2024.7.0: +viz", type=("build", "run"))
     depends_on("py-seaborn", when="@2024.7.0: +viz", type=("build", "run"))
     depends_on("py-nc-time-axis", when="@2024.7.0: +viz", type=("build", "run"))
+    depends_on("py-cartopy@0.23:", when="@2025.7.1: +viz", type=("build", "run"))
+
+    # +etc
+    depends_on("py-sparse@0.15:", when="@2025.7.1: +etc", type=("build", "run"))
+
+    # +accel
+    depends_on("py-scipy@1.13:", when="@2025.7.1: +accel", type=("build", "run"))
+    depends_on("py-bottleneck", when="@2025.7.1: +accel", type=("build", "run"))
+    depends_on("py-numbagg@0.8:", when="@2025.7.1: +accel", type=("build", "run"))
+    depends_on("py-numba@0.59:", when="@2025.7.1: +accel", type=("build", "run"))
+    depends_on("py-flox@0.9:", when="@2025.7.1: +accel", type=("build", "run"))
+    depends_on("py-opt-einsum", when="@2025.7.1: +accel", type=("build", "run"))
+
+    # https://github.com/pydata/xarray/issues/10588
```

Comment:
Commented on this, I think you just need a newer version of setuptools

Reply:
I think you're right. Double checking

Reply:
yup thanks